sql/database.sql

```diff
@@ -9938,13 +9938,148 @@ CREATE TABLE `form_groups_encounter` (
 
 DROP TABLE IF EXISTS `form_therapy_groups_attendance`;
 CREATE TABLE `form_therapy_groups_attendance` (
-  id	bigint(20) auto_increment,
-  date	date,
-  group_id	int(11),
-  user	varchar(255),
-  groupname	varchar(255),
-  authorized	tinyint(4),
-  encounter_id	int(11),
-  activity	tinyint(4),
+  id    bigint(20) auto_increment,
+  date  date,
+  group_id  int(11),
+  user  varchar(255),
+  groupname varchar(255),
+  authorized    tinyint(4),
+  encounter_id  int(11),
+  activity  tinyint(4),
   PRIMARY KEY (`id`)
 ) ENGINE=InnoDB ;
+--
+-- Table structure for table `medex_icons`
+--
+
+CREATE TABLE IF NOT EXISTS `medex_icons` (
+  `i_UID` int(11) NOT NULL AUTO_INCREMENT,
+  `msg_type` varchar(50) NOT NULL,
+  `msg_status` varchar(10) NOT NULL,
+  `i_description` varchar(255) NOT NULL,
+  `i_html` text NOT NULL,
+  `i_blob` longtext NOT NULL,
+  PRIMARY KEY (`i_UID`)
+) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=37 ;
```

Comment:
Regarding AUTO_INCREMENT, see above database.sql examples to see what to see what to set this to in your table.